File: app/src/main/java/com/hinterlong/kevin/swishticker/data/Team.java
package com.hinterlong.kevin.swishticker.data;

import java.util.ArrayList;
import java.util.List;

public class Team {
    private String name;
    private List<Integer> playerIds;

    public Team(String name) {
        this.name = name;
        playerIds = new ArrayList<>();
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public void addPlayer(int playerId) {
        playerIds.add(playerId);
    }

    public void removePlayer(int playerId) {
        playerIds.remove(Integer.valueOf(playerId));
    }

    public List<Integer> getPlayers() {
        return playerIds;
    }
}

File: app/src/main/java/com/hinterlong/kevin/swishticker/data/Team.kt
package com.hinterlong.kevin.swishticker.data

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(
    tableName = "teams"
)
data class Team(
    @ColumnInfo(name = "name") var name: String,
    @ColumnInfo(name = "color") var color: String? = null
){
    @PrimaryKey(autoGenerate = true)
    @ColumnInfo(name = "id")
    var id: Long = 0
}